README.md

```diff
@@ -20,153 +20,44 @@ analysis of the different steps and combinations of preprocesssing and integrati
 
 ### Please cite:
 
-_**Benchmarking atlas-level data integration in single-cell genomics.**  
-MD Luecken, M Büttner, K Chaichoompu, A Danese, M Interlandi, MF Mueller, DC Strobl, L Zappia, M Dugas, M Colomé-Tatché, FJ Theis
-bioRxiv 2020.05.22.111161; doi: https://doi.org/10.1101/2020.05.22.111161 _
+Luecken, M.D., Büttner, M., Chaichoompu, K. et al. Benchmarking atlas-level data integration in single-cell genomics.
+Nat Methods 19, 41–50 (2022). https://doi.org/10.1038/s41592-021-01336-8
 
 ## Installation
 
-To reproduce the results from this study, three different conda environments are needed.
-There are different environments for the python integration methods, the R integration methods and
-the conversion of R data types to anndata objects.
+To reproduce the results from this study, two separate conda environments are needed for python and R operations.
+Please make sure you have [`conda`](https://conda.io/projects/conda) installed on your system to be
+able to use the pipeline.
+We also recommend installing [`mamba`](https://mamba.readthedocs.io) for shorter installation times and smaller memory
```

Comment:
Can't we just recommend mambaforge here? That's one less thing to install and has a osx-arm native version https://github.com/conda-forge/miniforge

Reply:
I added mambaforge as an additional option. Many users probably already have conda, so it would be easier to just install mamba on top.